crates/swc_ecma_transforms_compat/src/es2015/arrow.rs

```diff
@@ -69,6 +70,7 @@ struct Arrow {
 impl VisitMut for Arrow {
     noop_visit_mut_type!();
 
+    #[tracing::instrument(skip_all)]
```

Comment:
Instead of manually creating span or use span! macro with hardcoded name, we'll use `#instrument` attr in general. We can always use span! or manual span for the cases we need additional customization but as a first step decorating all of our overridden visitor with default span would makes sense.